pages/settings/style.css

```diff
@@ -23,18 +23,16 @@ h1 {
   position: relative;
   display: flex;
   width: 100%;
-  padding: 1rem;
   align-items: center;
   gap: 0.5rem;
+  padding: 0.5rem 1rem;
   border: none;
   border-radius: 1rem;
-  font-size: var(--body-1-large-size);
-  letter-spacing: var(--body-1-large-letter-spacing);
-  line-height: var(--body-1-large-line-height);
-  font-weight: 500;
-  color: var(--neutral-neutral-900, #101928);
```

Comment:
replacing color variables from
  color: var(--neutral-neutral-900, #101928);
  background: var(--secondary-secondary-200, #edf4f2);

to

 color: var(--neutral-900);
  background: var(--secondary-200);